mne/beamformer/_compute_beamformer.py

```diff
@@ -427,7 +427,7 @@ def _compute_beamformer(G, Cm, reg, n_orient, weight_norm, pick_ori,
             # Estimate noise level based on covariance matrix, taking the
             # first eigenvalue that falls outside the signal subspace or the
             # loading factor used during regularization, whichever is largest.
-            if rank >= len(Cm):
+            if rank > len(Cm):
                 # Covariance matrix is full rank, no noise subspace!
                 # Use the loading factor as noise ceiling.
                 if loading_factor == 0:
```

Comment:
The error message should be changed. First, it suggests that it is not possible to compute a noise level with full-rank cov matrix (also see comment in code above). Second, now this would only be reached if the user was specifying the rank manually _already_, telling them to do the same thing. 
If we want to keep this case I suggest something like:
```'Cannot compute noise level for NAI computation since specified rank (%s) is higher than covariance matrix dimensions (%s) and no regularization is specified. Try changing rank to equal the covariance matrix dimensions.' % (rank, len(Cm))```

Reply:
Seems reasonable to me. @wmvanvliet this is your code I think, thoughts?

Reply:
It works on the data now.

Reply:
how can the rank of the data be higher than the number of dimensions of the data?

Reply:
Well, good question. I think it is now a remainder of the code how it was before (  ```if rank >= len(Cm)```  ). As I mentioned above, I do not see an immediate use case for it - except the user specifying a rank manually, maybe in a combination with dropping channels during the filter computation. 
Do we all agree then that the ```if``` statement can go and we go back to how the noise was computed before for any case? In our experience that worked fine, and is in accordance with what Fieldtrip does, too.

Reply:
Okay with me